exporters/prometheus/exporter.go

```diff
@@ -34,38 +37,50 @@ import (
 type Exporter struct {
 	metric.Reader
 	Collector prometheus.Collector
```

Comment:
Does this need to be exported still?

Reply:
No, but it is currently used by the collector.  I will attempt to deprecate it, and we can remove it in the next round to allow for a clean migration.

Reply:
I changed my mind, with New() changes users will have to update the code regardless.  I will just unexport it.